index.js

```diff
@@ -29,7 +29,9 @@ class HyperDHT extends DHT {
     this.defaultKeyPair = opts.keyPair || createKeyPair(opts.seed)
     this.listening = new Set()
     this.tracer = createTracer(this)
-    this.connectionKeepAlive = opts.connectionKeepAlive || 0
+    this.connectionKeepAlive = opts.connectionKeepAlive == null
```

Comment:
This still supports passing in a 0 value. We can also prohibit a 0 value by doing `this.connectionKeepAlive = opts.connectionKeepAlive || 5000`

Reply:
Cleaned up the contract to `opts.connectionKeepAlive === false ? 0 : opts.connectionKeepAlive || 5000`